examples/src/main/java/com/example/JpegImageParserFuzzer.java

```diff
@@ -22,6 +22,16 @@
 
 // Found https://issues.apache.org/jira/browse/IMAGING-275.
 public class JpegImageParserFuzzer {
+  public static void fuzzerInitialize() {
+    // Only used to verify that arguments are correctly passed down to child processes.
+    if (System.getProperty("foo") == null || System.getProperty("bar") == null) {
+      // The child process did not have a sufficiently high memory limit,
```

Comment:
Is this comment relevant here?

Reply:
Good catch, it's a left over from a previous version of this check. I will remove it.